operator-common/src/main/java/io/strimzi/operator/common/model/Labels.java

```diff
@@ -178,6 +186,33 @@ public Labels withKind(String kind) {
     public Labels withCluster(String cluster) {
         return with(STRIMZI_CLUSTER_LABEL, cluster);
     }
+    
+    /**
+     * The same labels as this instance, but with the application name {@code strimzi} for the {@code app.kubernetes.io/name} key.
+     * @return A new instance with the given kubernetes application name added.
+     */
+    public Labels withKubernetesName() {
+        return with(Labels.KUBERNETES_NAME_LABEL, Labels.KUBERNETES_NAME);
+    }
+
+    /**
+     * The same labels as this instance, but with the given {@code instance} for the {@code app.kubernetes.io/instance} key.
+     * @param instance The instance to add.
+     * @return A new instance with the given kubernetes application instance added.
+     */
+    public Labels withKubernetesInstance(String instance) {
+        return with(Labels.KUBERNETES_INSTANCE_LABEL, instance);
+    }
+
+    /**
+     * The same labels as this instance, but with the given {@code operatorName} for the {@code app.kubernetes.io/managed-by} key.
+     * @param operatorName The name of the operator managing this resource.
+     * @return A new instance with the given operator that is managing this resourse.
+     */
+    public Labels withKubernetesManagedBy(String operatorName) {
+        // Make configurable?
```

Comment:
what does this comment mean? should we remove it, or it raises idea for an improvement?

Reply:
Lol, that was mid-development comment. From when I thought that the `Labels` class was used by all the operators (including user and topic operator).
Since user and topic operators have their own implementations of labels this does not need to be configurable, but it does make me wonder, is there any reason why we can't pull the Labels class into operator-common and have all three operator types use the same labels class/utilities?

Reply:
I can remove it in this PR, or I can remove it in my followup user/topic operator changes.

Delta can be found here if interested: https://github.com/strimzi/strimzi-kafka-operator/commit/f853383080fa6baa814ad706546780f86b08f5fc

Hoping to get it finished soon, but struggling with the Topic Operator as the session tests keep timing out when I add the labels!

Reply:
Good point about having the `Labels` class common across operators. For now, it's enough just removing the comment and we are ready to merge :-)